src/azure-cli/azure/cli/command_modules/synapse/operations/accesscontrol.py

```diff
@@ -4,55 +4,67 @@
 # --------------------------------------------------------------------------------------------
 
 from knack.util import CLIError
-from azure.synapse.accesscontrol.models import RoleAssignmentOptions
 from azure.cli.core.util import is_guid
 from azure.graphrbac.models import GraphErrorException
 from msrestazure.azure_exceptions import CloudError
-from .._client_factory import cf_synapse_client_accesscontrol_factory, cf_graph_client_factory
+from .._client_factory import cf_synapse_role_assignments, cf_synapse_role_definitions, cf_graph_client_factory
+from ..constant import ITEM_NAME_MAPPING
 
 
 # List Synapse Role Assignment
-def list_role_assignments(cmd, workspace_name, role=None, assignee=None):
-    # get role id
-    role_id = _resolve_role_id(cmd, role, workspace_name)
-    # get object_id
-    object_id = _resolve_object_id(cmd, assignee, fallback_to_object_id=True)
+def list_role_assignments(cmd, workspace_name, role=None, assignee=None, assignee_object_id=None,
+                          scope=None, item=None, item_type=None):
+    if bool(assignee) and bool(assignee_object_id):
+        raise CLIError('usage error: --assignee STRING | --assignee-object-id GUID')
```

Comment:
Consider the following error type
https://github.com/Azure/azure-cli/blob/3bff786b9e337aa35bb4cc64a09e5600a35663f8/src/azure-cli-core/azure/cli/core/azclierror.py#L152-L154

Reply:
Fixed